extensions/amp-subscriptions/0.1/crypto-handler.js

```diff
@@ -92,11 +92,36 @@ export class CryptoHandler {
    * @return {Promise<string>}
    */
   decryptDocumentContent_(
-    // eslint-disable-next-line no-unused-vars
     encryptedContent, documentKey) {
-    // Don't really return this. Placeholder for the real thing.
-    // const placeholder = encryptedContent.trim() + documentKey;
-    const placeholder = '<h2><i> abc </i></h2>';
-    return Promise.resolve(placeholder);
+
+console.log('encryptedContent', encryptedContent);
+console.log('documentKey', documentKey);
+
+    const decryptedContent = crypto.subtle.decrypt(
+        {
+          name: 'AES-CTR',
+          counter: new Uint8Array(16), // iv: all zeros.
+          length: 128, // block size (16): 1-128
+        },
+        this.base64ToBytes(documentKey),
+        encryptedContent.trim(),
+    ).then(function(buffer) {
+      return new TextDecoder().decode(new Uint8Array(buffer));
+    });
+    return Promise.resolve('<h2><i>' + decryptedContent + '</i></h2>');
```

Comment:
Let's remove `<h2><i>` now. If you'd like, you can add it to the encrypted content itself.